README.md

```diff
@@ -149,6 +149,22 @@ If both these parameters are present, the node will also start service SSL traff
       "Config": { // configuration passed to the binary inside the container
         "api": "v1"
       }
+    },
+    "custom-service": {
```

Comment:
if this is a template for any concrete service with a concrete name I'd prefer `a-service-name` or `service-name`

Reply:
done